api.js

```diff
@@ -40,23 +40,17 @@ util.inherits(Api, EventEmitter);
 module.exports = Api;
 
 Api.prototype._runFile = function (file) {
-	var args = [file];
+	var opts = {
+		file: file,
+		failFast: this.failFast,
+		serial: this.serial,
```

Comment:
I think we should do `this.options = options` in the `Api` constructor so we can just pass the whole options object here instead of explicitly specify each option.